.circleci/config.yml

```diff
@@ -0,0 +1,135 @@
+version: 2
+
+job-setup: &job-setup
+  docker:
+    - image: circleci/node:8.11.3-browsers
+  working_directory: ~/repo
+
+restore-cache: &restore-cache
+  keys:
+    - v1-dependencies-{{ checksum "package.json" }}
+    # fallback to using the latest cache if no exact match is found
+    - v1-dependencies-
+
+save-cache: &save-cache
+  paths:
+    - node_modules
+    - ~/.cache
+  key: v1-dependencies-{{ checksum "package.json" }}
+
+install-bolt: &install-bolt
+  - run:
+      name: Install Bolt
+      command: sudo yarn global add bolt
+
+jobs:
+  install:
+    <<: *job-setup
+    steps:
+      # Fetch code
+      - checkout
+
+      # Download and cache dependencies
+      - restore_cache: *restore-cache
+
+      - run:
+          name: Install Bolt
+          command: sudo yarn global add bolt
+
+      - run:
+          name: Install Dependencies
+          command: bolt
+
+      - run:
+          name: Install Cypress
+          command: node_modules/.bin/cypress install
+
+      - save_cache: *save-cache
+
+  lint:
+    <<: *job-setup
+    steps:
+      - checkout
+
+      - restore_cache: *restore-cache
+
+      - run:
+          name: Install Bolt
```

Comment:
Is this not covered by the `install` job?

Reply:
The caching wasn't working before so I was installing dependencies in each job independently. I'll try removing it now and see what happens